### PR TITLE
NAS-122951 / 23.10 / Adjust http warning to appear on login screen

### PR DESCRIPTION
**Pull request by AlexKarpov98**
Testing: check ticket
You should see:
( I tried to be consistent with the current design )

Screenshot 2023-07-21 at 16 12 05

### CODE REVIEW

File: src/app/views/sessions/signin/snackbar/insecure-connection.component.html [hunk 1]
```diff
@@ -1,8 +1,4 @@
 <ix-icon name="info"></ix-icon>
-<span class="message">{{ 'You are using an insecure connection. Switch to HTTPS for secure access.' | translate }}</span>
-<button mat-button ixTest="configure" (click)="onConfigure()">
-  <span class="text-configure">{{ 'Configure SSL' | translate }}</span>
-</button>
-<button mat-icon-button ixTest="close" (click)="onClose()">
-  <ix-icon name="close"></ix-icon>
-</button>
+<span class="message">
+  {{ 'You are using an insecure connection. Switch to HTTPS for secure access.' | translate }}
```
Comment (RehanY147) [inline src/app/views/sessions/signin/snackbar/insecure-connection.component.html:3]: Let's add a link to `Switch to HTTPs` string so it's clickable with the https link redirection

**Review by RehanY147 — CHANGES_REQUESTED** [review]

File: src/app/views/sessions/signin/snackbar/insecure-connection.component.html (hunk 1 above)
Comment (AlexKarpov98) [reply src/app/views/sessions/signin/snackbar/insecure-connection.component.html:3]: > Let's add a link to `Switch to HTTPs` string so it's clickable with the https link redirection

How will user see the FORM if he's not logged in yet❓

Comment (RehanY147) [reply src/app/views/sessions/signin/snackbar/insecure-connection.component.html:3]: I don't understand what you mean. I'm saying make it something like
`'You are using an insecure connection. <a href="https://naslink.com">Switch to HTTPS</a> for secure access.'`

**Review by RehanY147 — CHANGES_REQUESTED** [review]
I don't understand what you mean. I'm saying make it something like
'You are using an insecure connection. <a href="https://naslink.com">Switch to HTTPS</a> for secure access.'

File: src/app/views/sessions/signin/snackbar/insecure-connection.component.html (hunk 1 above)
Comment (AlexKarpov98) [reply src/app/views/sessions/signin/snackbar/insecure-connection.component.html:3]: Oh, I see.
Which link should be there?
Before it worked like - If I click a LINK - sidenav opened with a form, I thought you mean open form again.

**Review by RehanY147 — APPROVED** [review]

**Pull request merged**